src/main/java/org/threadly/litesockets/SocketExecuterCommonBase.java

```diff
@@ -145,16 +144,13 @@ public void stopListening(final Server server) {
     if(!checkServer(server)) {
       return;
     } else {
-      if(server.getServerType() == WireProtocol.TCP) {
+      if(server instanceof TCPServer) {
```

Comment:
The above check should have been more efficient than this now instanceof check.  Why the change?